app/models/ahoy/event.rb

```diff
@@ -0,0 +1,8 @@
+class Ahoy::Event < ApplicationRecord
```

Comment:
Since we do a lot of tracking and reporting in Datadog, down the road you might want to hook into this model on creation and send the stats to Datadog as well.

Reply:
@mstruve What sorts of things do you think would be useful for Datadog reporting as opposed to just tracking? Trying to think through what/when we'd want to report (if there's something obvious, I can add it to the project board to tackle)

Reply:
I was thinking just basic counts to start
```ruby
DatadogStatsClient.count("welcome_notification_clicks", <number>, tags: ["title:<the title>"])
```
That would allow you to graph over time the number of clicks for each different type of welcome notification and compare them all to each other